### PR TITLE
feat: updated one-click deploys

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

- Added `railway.toml` to deploy from `/docker/Dockerfile`
- Added Koyeb as a deploy option

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -280,12 +280,16 @@ WantedBy=multi-user.target
 
 ### Railway
 
-[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/DjrRRX)
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/bG6D4p?referralCode=OEa9MT)
```
Comment (catalinpit) [inline README.md:283]: Shouldn't this simply be `https://railway.app/template/bG6D4p`?

Comment (adithyaakrishna) [inline README.md:283]:
```suggestion
[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/bG6D4p)
```

Comment (adithyaakrishna) [reply README.md]: Yes Yes, updated it just now :)

**Review by Mythie — APPROVED** [review]

**Comment by adithyaakrishna** [comment]
Bravo 6 Going Dark

**Pull request merged**